ee/codegen/src/__test__/helpers/node-data-factories.ts

```diff
@@ -28,9 +28,9 @@ export function entrypointNodeDataFactory(): EntrypointNode {
   };
 }
 
-export function terminalNodeDataFactory(): FinalOutputNode {
+export function terminalNodeDataFactory(id?: string): FinalOutputNode {
```

Comment:
nit: let's make this an optional set of kwargs from the get-go. Note: My syntax below might be wrong.

```suggestion
export function terminalNodeDataFactory({id}: {id?: string} = {}): FinalOutputNode {
```